resources/new_icons.yaml

```diff
@@ -18,3 +18,6 @@ wordbook:
 
 weather:
   - com.coloros.weather2/com.oplus.weather.main.view.WeatherMainActivity
+
+meetup_for_organizers:
+  - com.meetup/com.meetup.organizer
```

Comment:
I'm not 100% this is correct

Reply:
It's com.meetup.organizer/com.meetup.organizer.MainActivity

Reply:
Fixed, thanks!